app/src/main/java/com/swent/skillswap/ui/profile/ProfileScreen.kt

```diff
@@ -41,196 +56,320 @@ fun ProfileScreen(
-                            indication = null
-                        ) {
-                            onAccountReviewClick()
-                        }
-                        .padding(start = 28.dp, bottom = 12.dp)
-                )
+                    .padding(vertical = 12.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(
+                text = title,
+                fontSize = 14.sp,
+                color = Color.White,
+                fontWeight = FontWeight.Medium,
+                modifier = Modifier.weight(1f)
+            )
 
-                // Horizontal white line
-                Canvas(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .height(1.dp)
-                ) {
-                    drawLine(
-                        color = Color.White,
-                        start = androidx.compose.ui.geometry.Offset(0f, 0f),
-                        end = androidx.compose.ui.geometry.Offset(size.width, 0f),
-                        strokeWidth = 1.dp.toPx(),
-                        cap = StrokeCap.Round
-                    )
-                }
-
-                Spacer(modifier = Modifier.height(12.dp))
-
-                // Add more accounts button (clickable)
-                Text(
-                    text = "Add more accounts",
-                    fontSize = 14.sp,
-                    color = Color.White,
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .clickable(
-                            interactionSource = remember { MutableInteractionSource() },
-                            indication = null
-                        ) {
-                            onAddAccountsClick()
-                        }
-                        .padding(vertical = 12.dp)
-                )
-            }
+            Icon(
+                imageVector =
+                    if (isExpanded) Icons.Default.KeyboardArrowUp
+                    else Icons.Default.KeyboardArrowDown,
+                contentDescription = if (isExpanded) "Collapse" else "Expand",
+                tint = Color.White,
+                modifier = Modifier.size(20.dp)
+            )
         }
+
+        // Expandable content
+        AnimatedVisibility(
+            visible = isExpanded,
+            enter = expandVertically(animationSpec = tween(300)),
+            exit = shrinkVertically(animationSpec = tween(300))
+        ) {
+            content()
+        }
+    }
+}
+
+@Composable
+fun HorizontalDivider() {
+    Canvas(modifier = Modifier.fillMaxWidth().height(1.dp)) {
+        drawLine(
+            color = ProfileDivider,
+            start = Offset(0f, 0f),
+            end = Offset(size.width, 0f),
+            strokeWidth = 1.dp.toPx(),
+            cap = StrokeCap.Round
+        )
     }
-}
\ No newline at end of file
+}
```

Comment:
This is redundant code, either remove the ProfileDivider.kt file as a whole or change the HorizontalDivider to a call for ProfileDivider. Watch out for imports (for ProfileDivider), the you didn't import the right package so the code doesn't run